apps/frontend/src/app/5_pages/ProtocolDataPage/components/LendAndBorrow/LendAndBorrow.tsx

```diff
@@ -1,19 +1,111 @@
-import React, { FC } from 'react';
+import React, { FC, useCallback } from 'react';
 
 import { t } from 'i18next';
+import { useNavigate } from 'react-router-dom';
 
-import { Paragraph } from '@sovryn/ui';
+import {
+  Button,
+  ButtonStyle,
+  Paragraph,
+  SimpleTableRow,
+  Table,
+} from '@sovryn/ui';
 
+import { AssetRenderer } from '../../../../2_molecules/AssetRenderer/AssetRenderer';
+import { translations } from '../../../../../locales/i18n';
+import { LendingPool } from '../../../../../utils/LendingPool';
+import { LendingPoolDictionary } from '../../../../../utils/LendingPoolDictionary';
+import { normalizeToken } from '../../../BorrowPage/BorrowPage.utils';
+import { AvailableSupply } from '../../../BorrowPage/components/BorrowAssetsTable/components/AvailableSupply/AvailableSupply';
+import { NextBorrowInterestRate } from '../../../BorrowPage/components/BorrowAssetsTable/components/NextBorrowInterestRate/NextBorrowInterestRate';
+import { NextSupplyInterestRate } from '../../../LendPage/components/NextSupplyInterestRate/NextSupplyInterestRate';
 import { pageTranslations } from '../../ProtocolDataPage.constants';
+import { COLUMNS_CONFIG } from './LendAndBorrow.constants';
 
-export const LendAndBorrow: FC = () => (
-  <div className="w-full md:bg-gray-90 md:py-7 md:px-6 rounded mb-9">
-    <Paragraph
-      className="md:text-2xl text-base font-medium"
-      children={t(pageTranslations.sections.lendAndBorrow)}
-    />
-    <div className="min-h-72 mt-5">
-      <p>TODO: Add the actual content here</p>
+const lendPools = LendingPoolDictionary.list();
+
+export const LendAndBorrow: FC = () => {
+  const navigate = useNavigate();
+
+  const generateRowTitle = useCallback(
```

Comment:
on mobile row missed `Borrow APR` info, you can check how its implemented in `LendFrame`, its pretty the same

Reply:
Well on the design it's not showing the APR. and if you think about it it's not clear to show borrow or lend APR in title.
That's why i'm showing both values on the content